### DISCUSSION TITLE
Otomatik cevap oluşturmada sorun yaşıyorum ?

### DISCUSSION THREAD

**Discussion by SEOTurk** [VolkanSah/aicc-comments-reply#2] [General]
![Screenshot 2023-05-09 at 11-01-42 Comments ‹ Wordpress — WordPress](https://user-images.githubusercontent.com/132989952/237032322-dba8d3bc-bb5c-486e-92b4-2e31a7a8c657.png)

Volkan selamlar, öncelikle eklenti için çok teşekkürler.
Bugün deneme fırsatım oldu ama bir türlü çalıştıramadım :(
API key oluşturdum, örnek bir yorum bıraktım.

Ekran görüntüsü ilettim, API key ile mi sorun yaşıyorum acaba ? Sunucuda olması gereken veya kapatılması gereken birşey var mı ?

İyi çalışmalar.

**Comment by VolkanSah** [comment]
Merhaba! WordPress için yorum yanıtı eklentimde sorun yaşadığınızı duymak beni üzüyor. Eklentinin çalışmamasının birkaç nedeni olabilir. İşte deneyebileceğiniz bazı adımlar:

OpenAI API anahtarlarınızı kontrol edin ve doğru olduğundan ve eklentinin seçenekler sayfasına doğru şekilde girildiğinden emin olun.

İstek-API URL'sinin doğru olduğundan ve sözdiziminde herhangi bir hata olmadığından emin olun.

Bir güvenlik yazılımının eklentinin çalışmasını engelleyip engellemediğini kontrol edin.

Hosting sağlayıcınızın OpenAI API isteklerini engelleyip engellemediğini kontrol edin.

Hataları aramak için konsolu kontrol edin ve size daha iyi yardımcı olabilmem için ilgili hata kodunu bana iletin.

Eğer hala sorun yaşıyorsanız, bana istediğiniz zaman ulaşabilirsiniz. Sorunu çözmenize yardımcı olmak için elimden geleni yapacağım. Teşekkür ederim!
